Makefile

```diff
@@ -0,0 +1,66 @@
+# TODO: run golint and errcheck, but only to catch *new* violations and
+# decide whether to change code or not (e.g. we need to be able to whitelist
+# violations already in the code). They can be useful to catch errors, but
+# they are just too noisy to be a requirement for a CI -- we don't even *want*
+# to fix some of the things they consider to be violations.
+.PHONY: ci
+ci: deps checkgofmt vet staticcheck unused ineffassign predeclared test
+
+.PHONY: deps
+deps:
+	go get -d -v -t ./...
+
+.PHONY: updatedeps
+updatedeps:
+	go get -d -v -t -u -f ./...
+
+.PHONY: install
+install:
+	go install ./...
+
+.PHONY: checkgofmt
+checkgofmt:
+	gofmt -s -l .
```

Comment:
what's the first call to `gofmt` for?

Reply:
To actually print the violations to the test output, so the user can see what files failed the check.